docs/migration-guide.rst

```diff
@@ -23,12 +23,14 @@ Rasa 1.2 to Rasa 1.3
 
 General
 ~~~~~~~
-
+- Default parameters and architectures for both ``EmbeddingPolicy`` and
+  ``EmbeddingIntentClassifier`` are changed. **Old trained models cannot be loaded**.
+  You need to retrain your models if you used these algorithms.
```

Comment:
you need to retrain in any case - all models will be marked as incompatible

Reply:
I meant if you use spacy+keras you're fine

Reply:
aaaah, because of minimum compatibility, I see